EIPS/eip-BN256_HashToCurve.md

```diff
@@ -0,0 +1,342 @@
+---
+eip: TBD
+title: Precompile for BN256 HashToCurve Algorithms
+author: Dr. Christopher Gorman, MadHive (@chgormanMH)
+discussions-to: https://ethereum-magicians.org/t/pre-compile-for-bls/3973
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-10-23
+requires: EIP-198, EIP-1108
+---
+
+## Simple Summary
+
+This EIP opens up opportunities for greater use of BN256
+elliptic curve operations by adding a precompiled contract for a
+deterministic hash-to-curve algorithm.
+In particular, this enables BLS signature verification in the EVM
+and reduces its cost to essentially that of one pairing check.
+
+There is a similar proposal to add precompiled contracts for the
+elliptic curve BLS12-381 in
+[EIP-2537](./eip-2537.md);
+this proposal includes a deterministic map-to-curve algorithm.
```

Comment:
Simple Summary should be much shorter.  Imagine an email subject line, or a forum sub-title, or a news headline or sub-headline.  Usually about a sentence is good, and often that sentence will be particularly terse and maybe not even grammatically correct!

It should not go into the "why" for the EIP, just describe in very pithy terms what this EIP is.  In this case, something like this may be good? (you know the matter better than I do to evaluate what a good headline would be):
```suggestion
Defines a hash-to-curve precompile for use in BLS12-381 ECC.
```

Reply:
Simple Summary has been simplified.